elementor/elementor
# Fix: CSS fails “Ensure text remains visible during webfont load” audit by Google’s PageSpeed Insights

## PR Checklist



## PR Type
What kind of change does this PR introduce?


## Summary

This PR can be summarized in the following changelog entry:

* Fix failed “Ensure text remains visible during webfont load” audit by Google’s PageSpeed Insights

## Description
An explanation of what is done in this PR

* Added "font-display: swap;" to relevant CSS classes.

## Test instructions
This PR can be tested by following these steps:

* Use Google PageSpeed to analyze a website using Elementor. (https://pagespeed.web.dev/)
* Check the "Performance" tab
* Check the sources of the "Ensure text remains visible during webfont load" error
* One of the sources will be /wp-content/plugins/elementor/assets/lib/eicons/css/elementor-icons.css

## Quality assurance


Fixes #
